README.md

```diff
@@ -1,14 +1,14 @@
 # Windows Package Manager Community Repository
-[![Gitter](https://img.shields.io/badge/chat-on%20gitter-yellow.svg)](https://gitter.im/Microsoft/winget-pkgs)
 
 This repository contains the manifest files for the **Windows Package Manager** default source.  You are highly encouraged to submit manifests for your favorite application.
->Note: At this time installers must either be MSIX, MSI or .exe application installers. Standalone or portable executables, compressed .zip files, and fonts are not currently supported.
+> Note: At this time installers must either be MSIX, MSI or .exe application installers. Standalone or portable executables, compressed .zip files, and fonts are not currently supported.
 
 The **Windows Package Manager** is an [open source client](https://github.com/microsoft/winget-cli) designed for command-line usage.
 
+[![Gitter](https://img.shields.io/gitter/room/Microsoft/winget-pkgs?style=for-the-badge)](https://gitter.im/Microsoft/winget-pkgs)
```

Comment:
I like the badge that was there :( 

Is there a reason it needs to change?

Reply:
I feel that it would be better there. The badge doesn't need to be above the description of the repository.

Reply:
It seems very rare to me that people put badges not at the top of the README. I've never actually seen it except here, it's designed so you can see them without scrolling. That's why the images are usually less than 100x100.